config/Config-images.in

```diff
@@ -16,6 +16,7 @@ menu "Target Images"
 			prompt "Compression"
 			default TARGET_INITRAMFS_COMPRESSION_LZMA if TARGET_apm821xx
 			default TARGET_INITRAMFS_COMPRESSION_LZMA if TARGET_ar71xx
+			default TARGET_INITRAMFS_COMPRESSION_LZMA if TARGET_ath79
```

Comment:
Will that affect other devices?

Reply:
Yes, and I posted a question to the email list regarding this here - https://lists.infradead.org/pipermail/openwrt-devel/2020-February/021923.html.

Without this, the compiled image won't boot. (I suspect something else in the lzma-loader needs to change to enable booting images larger than ~8MB on this device, so I will have a play with the settings.)

Basically "uncompressed size: 8396454 bytes" will boot, anything much larger will fail.